packages/mobile/src/escrow/saga.ts

```diff
@@ -178,6 +181,7 @@ function* withdrawFromEscrow() {
     Logger.showMessage(i18n.t('inviteFlow11:transferDollarsToAccount'))
   } catch (e) {
     Logger.error(TAG + '@withdrawFromEscrow', 'Error withdrawing payment from escrow', e)
+    CeloAnalytics.track(CustomEventNames.escrow_failed_to_withdraw, { error: e })
```

Comment:
Sometimes we pass through `e` and sometimes `e.message`, what should be the general default?

Reply:
Switched over to e.message consistently, this should be the default to avoid an unexpected large error in the analytics